### PR TITLE
feat: twitter langchain refinements

### PR DESCRIPTION
**Pull request by stat**
### What changed? Why?
- Refactor `twitter-langchain` to align with the package/action structure of `cdp-langchain`

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: twitter-langchain/twitter_langchain/twitter_api_wrapper.py [hunk 1]
```diff
@@ -46,34 +44,14 @@ def validate_environment(cls, values: dict) -> Any:
 
         return values
 
-    def account_details_wrapper(self) -> str:
-        """Get the authenticated Twitter (X) user account details.
+    def run_action(self, func: Callable[..., str], **kwargs) -> str:
+        """Run a Twitter Action."""
+        import tweepy
```
[the remark this answers is not in the document]
Comment (stat) [reply twitter-langchain/twitter_langchain/twitter_api_wrapper.py]: i removed the tweepy package check during validation and moved the import to the top

File: twitter-langchain/twitter_langchain/twitter_api_wrapper.py [hunk 2]
```diff
@@ -24,13 +23,6 @@ def validate_environment(cls, values: dict) -> Any:
...
         access_token_secret = get_from_dict_or_env(values, "twitter_access_token_secret", "TWITTER_ACCESS_TOKEN_SECRET")
 
-        try:
-            import tweepy
-        except Exception:
-            raise ImportError(
-                "Tweepy Twitter SDK is not installed. " "Please install it with `pip install tweepy`"
-            ) from None
-
         client = tweepy.Client(
```
Comment (John-peterson-coinbase) [inline twitter-langchain/twitter_langchain/twitter_api_wrapper.py:26]: this validation should stay I think. The one I wanted to remove was the extra import in `run_action`

Comment (John-peterson-coinbase) [reply twitter-langchain/twitter_langchain/twitter_api_wrapper.py]: resolved in most recent commit

**Review by jazz-cb — APPROVED** [review]

**Pull request merged**